test/e2e/k8s_util.go

```diff
@@ -124,12 +126,11 @@ func (k *KubernetesUtils) Probe(ns1, pod1, ns2, pod2 string, port int32, protoco
 		// log this error as trace since may be an expected failure
 		log.Tracef("%s/%s -> %s/%s: error when running command: err - %v /// stdout - %s /// stderr - %s", ns1, pod1, ns2, pod2, err, stdout, stderr)
 		// do not return an error
-		if protocol == v1.ProtocolTCP && strings.Contains(stderr, TCPRejectProbeReturn) || protocol == v1.ProtocolUDP && strings.Contains(stderr, UDPRejectProbeReturn) {
-			return Rejected, nil
-		} else if strings.Contains(stderr, DropProbeReturn) {
+		if strings.Contains(stderr, "TIMEOUT") {
 			return Dropped, nil
+		} else {
+			return Rejected, nil
```

Comment:
nit: based on https://github.com/kubernetes/kubernetes/blob/master/test/images/agnhost/README.md we should use constants for `TIMEOUT` and `REFUSED` here? And we should check `strings.Contains(stderr, "REFUSED")` for REJECT?

Reply:
When we reject UDP cases, we return ICMP type 3 code 10 which will be `UNKNOWN: no route to host` in agnhost implementation. It won't be `REFUSED`. The detailed discussion you can check this issue #1967.

Reply:
make sense to me. Thanks for the explanation.